src/main/java/uk/ac/ebi/embl/api/validation/dao/EraproDAOUtilsImpl.java

```diff
@@ -114,113 +115,91 @@ public EraproDAOUtilsImpl (Connection connection)
 	{
 		this.connection=connection;
 	}
-	
+
+
+	private String getAddress(String analysisId) throws ValidationEngineException {
```

Comment:
- Suggestion to add comment:
Used to construct address for RL line from submission account details.
- Might call this fetchSubmitterAddress